### ISSUE TITLE
Linking failure in asm.js: Unexpected stdlib member

### ISSUE THREAD

**Issue by cduff** [diegomura/react-pdf#1627]
Introducing [@react-pdf/renderer](https://github.com/diegomura/react-pdf/) to my project has resulted in warning messages in my Jest test outputs, like:

```
(node:29756) V8: C:\project\node_modules\@react-pdf\yoga\src\build\Release\nbind.js:1781 Linking failure in asm.js: Unexpected stdlib member
```

I've created a minimal reproduction repo: https://github.com/cduff/react-pdf-linking-failure

Any idea how these can be resolved?

**Comment by cduff** [comment]
Could be related to https://github.com/facebook/yoga/issues/898 although the error message report by the OP there was a bit different.

**Comment by jeetiss** [comment]
Hi @cduff 

I think the warning is here from the [start of the time](https://github.com/diegomura/react-pdf/runs/4132958829?check_suite_focus=true#step:5:9)

It would be nice to fix this, but the error doesn't break anything, so it is not a priority now

**Comment by cduff** [comment]
> Hi @cduff
> 
> I think the warning is here from the [start of the time](https://github.com/diegomura/react-pdf/runs/4132958829?check_suite_focus=true#step:5:9)
> 
> It would be nice to fix this, but the error doesn't break anything, so it is not a priority now

Hi @jeetiss,

Yes, those warnings look related.

It's true that the error doesn't appear to break anything, but having Jest output polluted with error messages is quite annoying, e.g.

![image](https://user-images.githubusercontent.com/7855146/145119834-a9ccdcce-c5ba-4c8d-a2e8-fab1d7747748.png)

Therefore, I think it would be ideal for this to be resolved.

**Comment by chathu-novade** [comment]
Is there any update on this. I'm also getting this in jest tests.

**Comment by edmofro** [comment]
Has anyone fixed this, or at least found a way to silence that specific error to avoid spamming the test output?

**Comment by jeetiss** [comment]
fixed in #2105
